src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,68 @@
+package duke;
+
+import java.util.stream.Stream;
+
+/**
+ * A UI to show outputs, success and error messages to the users.
+ */
+public class Ui {
+    private void print(String s) {
+        String seperator = "    ____________________________________________________________\n";
+        System.out.println(seperator + Stream.of(s.split("\n")).map(x -> "     " + x + "\n")

```

Comment:
Interesting use of Streams here. Seems like a good addition as it is both intuitive and makes the code possibly less verbose